tests/RobotFramework/tests/cumulocity/log/log_operation.robot

```diff
@@ -92,6 +92,18 @@ Log file request not processed if operation is disabled for tedge-agent
     ...    expected_status=init
     ...    c8y_fragment=c8y_LogfileRequest
 
+Default plugin conifiguration
+    Set Device Context    ${DEVICE_SN}
+
+    # Remove the existing plugin configuration
+    Execute Command    rm /etc/tedge/plugins/tedge-log-plugin.toml
```

Comment:
Can this be why this test is failing to recreate the config?

```suggestion
    Execute Command    sudo rm /etc/tedge/plugins/tedge-log-plugin.toml
```

Reply:
I think it's just a race condition. After the agent is restarted, I need to wait till it publishes the supported log types message before doing the assertion. Else, the assertion is performed again the old values which are not updated yet.